MaintenanceDashboard.Data/API/RegisterToolContext.cs

```diff
@@ -2,87 +2,49 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using MaintenanceDashboard.Common;
+using MaintenanceDashboard.Data.Interfaces;
 
 namespace MaintenanceDashboard.Data.Api
 {
-    public sealed class RegisterToolContext : IDisposable, IRegisterToolContext
+    public sealed class RegisterToolContext : IRegisterToolContext
     {
-        private readonly DataContext context;
-
-        private bool disposed;
+        private readonly DataContext _context;
 
         public RegisterToolContext()
         {
-            context = new DataContext();
-        }
-
-        public DataContext DataContext
-        {
-            get { return context; }
+            _context = new DataContext();
         }
 
 
         public void CreateRegisterTool(RegisterTool registerTool)
         {
             CheckValue.RequireString(registerTool.ToolName);
 
-            context.RegisterTools.Add(registerTool);
-            context.SaveChanges();
+            _context.RegisterTools.Add(registerTool);
+            _context.SaveChanges();
         }
 
 
         public void UpdateRegisterTool(RegisterTool registerTool)
         {
-            var entity = context.RegisterTools.Find(registerTool.Id);
+            var entity = _context.RegisterTools.Find(registerTool.Id)
+                ?? throw new NotImplementedException(ErrorText.UNHANDLED_BY_API);
```

Comment:
usunąłem tego ifa na dole na rzecz operatora ternarnego `??`. Jeśli entity będzie nullem to wtedy będzie wyjątek. Dodałem też plik rsx ze stringami dla błędów, plik znajduje się w moim Commonie, zobacz sobie